x-pack/test/api_integration/apis/ml/data_frame_analytics/get_spaces.ts

```diff
@@ -140,5 +140,66 @@ export default ({ getService }: FtrProviderContext) => {
     it('should fail to list jobs stats by job ids if one of them is in a different space', async () => {
       await runRequest(idSpace1, 404, true, `${jobIdSpace1},${jobIdSpace2}`);
     });
+
+    describe('GetDataFrameAnalyticsIdMap with spaces', () => {
+      it(`should return a map of objects from ${idSpace1} leading up to analytics job id created in ${idSpace1}`, async () => {
+        const { body } = await supertest
+          .get(`/s/${idSpace1}/api/ml/data_frame/analytics/map/${jobIdSpace1}`)
```

Comment:
Is it worth moving these common steps out into a function similar to the `runRequest` we have for the existing tests in this file?

Reply:
Good call - moved in 381e99394ac6413c2ec4d5f71241f0981e5f2f5c